releasenotes/notes/update-bfgs-optimizer-29b4ffa6724fbf38.yaml

```diff
@@ -0,0 +1,7 @@
+upgrade:
+  - |
+    Deleted the deprecated ``factr`` and ``epsilon`` arguments of the :class:`.L_BFGS_B` optimizer.
+  - |
+    Change the default of maximum number of allowed function evaluations in :class:`.L_BFGS_B`
```

Comment:
Should we include `maxfun` name here to avoid people could put it in parentheses after the text e.g. "...allowed functional evaluations (`maxfun`) in ..."